src/macaron/util.py

```diff
@@ -126,7 +126,11 @@ def send_head_http_raw(
 
 
 def send_get_http_raw(
-    url: str, headers: dict | None = None, timeout: int | None = None, allow_redirects: bool = True
+    url: str,
+    headers: dict | None = None,
+    timeout: int | None = None,
+    allow_redirects: bool = True,
+    always_return_response: bool = False,
```

Comment:
The docstring misses the new parameter and the return value description needs to be adjusted.

Also, it seems more intuitive to change the parameter to 
```suggestion
   check_response_fail: bool = True,
```